views/pages/pathTraversal.gohtml

```diff
@@ -25,6 +25,18 @@
         </div>
         <button type="submit" class="btn btn-primary">Submit</button>
         </form>
+        <h4 class="sub-header">Buffered Query - <code>bytes.Buffer</code></h4>
```

Comment:
I think leaving it as `Buffered Query` instead of `Buffered Query - <code>bytes.Buffer</code>`.

So this way we are consistent with how it was done for xss `<h4 class="sub-header">Buffered Query</h4>`

Reply:
Agree with @Ms-Alex The previous section uses the code block before the sink name.